complete/src/main/java/com/example/restservice/service/smsService/MockSmsService.java

```diff
@@ -0,0 +1,13 @@
+package com.example.restservice.service.smsService;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("!prod")
+public class MockSmsService implements SmsService {
+    @Override
+    public String sendSMS(String contactNumber, String queueName) {
+        return "SUCCESS";
```

Comment:
Let's add a warning here: `log.warn("Mock SMS Sent");`.
Use @Slf4j logback annotation to get the loggeer.